seed/csharp-sdk/file-upload/src/SeedFileUpload/Service/ServiceClient.cs

```diff
@@ -41,11 +41,13 @@ public async Task JustFileAsync(JustFileRequet request)
 
     public async Task JustFileWithQueryParamsAsync(JustFileWithQueryParamsRequet request)
     {
-        var _query = new Dictionary<string, object>()
-        {
-            { "integer", request.Integer.ToString() },
-            { "listOfStrings", request.ListOfStrings },
-        };
+        var _query = new Dictionary<string, object>() { };
+        _query["integer"] = request.Integer.ToString();
+        _query["listOfStrings"] = request.ListOfStrings.Select(_value => _value).ToList();
```

Comment:
whats this bit doing?
`request.ListOfStrings.Select(_value => _value).ToList()`

Reply:
Nice catch, we should just use the list directly in this case. The implementation of the `Select` call depends on the result of the stringify method, which could yield different expressions like so:

```csharp
_query["operand"] = request
    .Operand.Select(_value => JsonSerializer.Serialize(_value))
    .ToList();
```

We'll just need to add in some context so we can recognize whether or not we actually need the `Select` call at all. I'll add that in here, too.

Reply:
will leave it up to you if this feels worth the lift!

Reply:
Definitely worth the lift - this actually ended up cleaning up quite a bit!